egs/mls_english/ASR/zipformer/decode.py

```
@@ -0,0 +1,1085 @@
+
+from icefall import ContextGraph, LmScorer, NgramLm
+from icefall.checkpoint import (
+    average_checkpoints,
+    average_checkpoints_with_averaged_model,
+    find_checkpoints,
+    load_checkpoint,
+)
+from icefall.lexicon import Lexicon
+from icefall.utils import (
+    AttributeDict,
+    make_pad_mask,
```

Ruff (0.12.2)

`icefall.utils.make_pad_mask` imported but unused

Remove unused import: `icefall.utils.make_pad_mask`

(F401)